test/reactTags.test.js

```diff
@@ -1,155 +1,189 @@
-import React from 'react';
-import { expect } from 'chai';
-import { mount } from 'enzyme';
-import { spy } from 'sinon';
-import noop from 'lodash/noop';
-import { WithContext as ReactTags } from '../lib/ReactTags';
+import React from "react";
+import { expect } from "chai";
+import { mount, shallow } from "enzyme";
+import { spy } from "sinon";
+import noop from "lodash/noop";
+import { WithContext as ReactTags } from "../lib/ReactTags";
+import renderer from "react-test-renderer";
 
 const defaults = {
   tags: [{ id: 1, text: "Apple" }],
   suggestions: ["Banana", "Apple", "Apricot", "Pear", "Peach"],
   handleAddition: noop,
   handleDelete: noop,
   handleDrag: noop
-}
+};
 
 const DOWN_ARROW_KEY_CODE = 40;
 
 function mockItem(overrides) {
   const props = Object.assign({}, defaults, overrides);
-  return <ReactTags {...props} />
+  return <ReactTags {...props} />;
 }
 
-describe("ReactTags", () => {
-  test("shows the classnames of children properly", () => {
-    const $el = mount(mockItem());
-    expect($el.find('.ReactTags__tags').length).to.equal(1);
-    expect($el.find('.ReactTags__selected').length).to.equal(1);
-    expect($el.find('.ReactTags__tagInput').length).to.equal(1);
-    expect($el.find('.ReactTags__tagInputField').length).to.equal(1);
-  });
+test("focus on input by default", () => {
+  const $el = mount(mockItem());
+  expect(document.activeElement.tagName).to.equal("INPUT");
+  expect(document.activeElement.className).to.equal("ReactTags__tagInputField");
+});
+
+test("should not focus on input if autofocus is false", () => {
+  const $el = mount(mockItem({autofocus: false}));
+  expect(document.activeElement.tagName).to.equal("BODY");
```

Comment:
I think it is not an issue with either of these. Normally it may not be an issue to have multiple ReactTags components mounted in a test environment. But since you are checking state that exists outside of the component and is common to all mounted components (`document`), then you will need to be careful to cleanup after your tests by unmounting. I don't think there is an expectation that enzyme will auto unmount after each test, or that jest or enzyme will cleanup the DOM after each test.